.github/sync-repo-settings.yaml

```diff
@@ -61,3 +61,5 @@ permissionRules:
     permission: admin
   - team: yoshi-java
     permission: push
+  - team: googleapis-auth
```

Comment:
I think this change might be overridden to this [file](https://github.com/googleapis/synthtool/blob/master/synthtool/gcp/templates/java_library/.github/sync-repo-settings.yaml) by the nightly owlbot template sync job, so we may need to add this file to the ignore list in `owlbot.py`. Is that correct @suztomo ?

Reply:
I just added Write role to googleapis-auth team via 'Settings'. Edit to .github/sync-repo-settings.yaml is not needed.

I observed OwlBot does not override the file https://github.com/googleapis/synthtool/pull/1494#discussion_r923910542